README.md

```diff
@@ -1,9 +1,201 @@
 # knuu
 
-This repository contains the source code for the knuu Integration Test Framework.
+[![CodeQL](https://github.com/celestiaorg/knuu/workflows/CodeQL/badge.svg)](https://github.com/celestiaorg/knuu/actions) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7475/badge)](https://bestpractices.coreinfrastructure.org/projects/7475)
 
-If you have feedback on the framework, want to report a bug or suggest an improvement, please create an issue.
+## Description
 
-## Example
+Knuu is an open-source project, and the goal of knuu is to provide a framework for writing integration tests.
+The framework is written in Go and is designed to be used in Go projects.
+The idea is to provide a framework that uses the power of containers and Kubernetes without the test writer having to know the details of how to use them.
 
-Check out the [knuu-example](https://github.com/celestiaorg/knuu-example) repository for an example of how to use the knuu Integration Test Framework.
+We invite you to explore our codebase, contribute, and join us in developing a framework to help projects write integration tests.
+
+## Features
+
+Knuu is designed around `Instances`, which you can create, start, control, communicate with other Instances, stop, and destroy.
+
+Some of the features of knuu are:
+
+- Initialize an Instance from a Container/Docker image
+- Configure startup commands
+- Configure Networking
+    - What ports to expose
+    - Disable networking to simulate network outages
+- Configure Storage
+- Execute Commands
+- Configure resources
+- Create a pool of Instances and control them as a group
+- And more to come...
```

Comment:
I would point to a global pinned issue to direct people somewhere more tangible

Reply:
What do you mean by that?

Reply:
You can create a high-level issue in knuu repo and pin it 
Then use the link from it and provide it here